apps/api-server/migrations/073-create-notification-messages.js

```diff
@@ -0,0 +1,72 @@
+const { Sequelize } = require('sequelize');
+
+module.exports = {
+
+  async up ({ context: queryInterface }) {
+    queryInterface.createTable('notification_messages', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
```

Comment:
Kunnen deze notificatie messages ook gevoelige gebruikersinformatie bevatten? Wellicht dat we dan het id i.p.v. number een UUID moeten maken? Dat de ids niet gegokt kunnen worden